### PR TITLE
Create Technical-writer.md

### PR DESCRIPTION
**Pull request by Shreyangi**

### CODE REVIEW

File: job-roles/Technical-writer.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Tecnical Writer
+At MayaData we believe that the best way to deliver storage and related services to containerized and cloud-
+native environments is with containerized and cloud-native architectures. Everything we do is focused on providing data agility and simplifying the daily operation of Kubernetes SRE’s and DevOps teams using 
+Kubernetes as a data layer. We enable our users to handle complex data management operations easily, and we have become open source leaders in building Kubernetes into a first-class data platform.
+
+### Requirements:
```
Comment (bobek) [inline job-roles/Technical-writer.md:6]:
```suggestion
## Requirements:
```

File: job-roles/Technical-writer.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+* SHOULD have some interest in technical marketing or technical communication
+* SHOULD have some understanding of SEO and related concepts
+* MUST have a sample of work (at least two articles or pieces of content) that they can share as part of a portfolio
+* MUST be willing to spend 30 minutes discussing the work with an interviewer
+* MUST be willing to learn about a technical subject that they’re unfamiliar with, and explain the subject back to their teacher
+* MUST be comfortable with morning meetings, in order to accommodate time zone differences
+* SHOULD be starting their career, or looking for a switch in careers
+* MUST have internet access, and be comfortable working at a remote-first company
+
+### Responsibilities:
```
Comment (bobek) [inline job-roles/Technical-writer.md:19]:
```suggestion
## Responsibilities:
```

**Review by bobek — COMMENTED** [review]
Can we please update the language to make it a more welcoming to people? Thx

**Comment by Shreyangi** [comment]
@kmova Can you please review this ?

File: job-roles/Technical-writer.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+* MUST be curious about people and technology and how people use technology to get things done
+* SHOULD have an interest in open source or software development
+* SHOULD have some interest in technical marketing or technical communication
+* SHOULD have some understanding of SEO and related concepts
+* MUST have a sample of work (at least two articles or pieces of content) that they can share as part of a portfolio
+* MUST be willing to spend 30 minutes discussing the work with an interviewer
+* MUST be willing to learn about a technical subject that they’re unfamiliar with, and explain the subject back to their teacher
+* MUST be comfortable with morning meetings, in order to accommodate time zone differences
+* SHOULD be starting their career, or looking for a switch in careers
+* MUST have internet access, and be comfortable working at a remote-first company
```
Comment (bobek) [inline job-roles/Technical-writer.md:17]: Please, change language to be closer to other job descriptions. We don't write RFC here and would not like to scare off potential candidates.

File: job-roles/Technical-writer.md [hunk 4]
```diff
@@ -0,0 +1,22 @@
+# Tecnical Writer
```
Comment (bobek) [inline job-roles/Technical-writer.md:1]:
```suggestion
# Technical Writer
```
Please run the doc through spell checker.

**Review by bobek — CHANGES_REQUESTED** [review]

**Comment by Shreyangi** [comment]
@bobek @kmova Can you please review the changes?

**Review by bobek — APPROVED** [review]

**Pull request merged**